README.md

```diff
@@ -92,6 +92,62 @@ PIWIK_UPDATE_START_GOAL_ID | Matomo Update Start goal ID | 3
 SHOW_SERVICE_OFFLINE_PAGE | Feature Flag | false
 VF01_FORM_DOWNLOAD_URL | Overseas entity verification checks statement URL | `https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/1095139/OE_VF01.pdf`
 
+
+## Terraform and variables required
+
+Terraform is run via a concourse ci pipeline and is used in place of AWS Cloud Formation for both provisioning and for deploying the cluster and services.
```

Comment:
Reads well, but I'm just curious where the new Terraform details came from?

Reply:
It's a mix of what is in the link referenced, what we have done so far with terraform and from what I have learned about AWS, Cloud Formation is AWS's own IoC tool for deployment.